udata_hydra/__init__.py

```diff
@@ -0,0 +1,37 @@
+import os
```

Comment:
After looking at the Ansible playbook, I think it will be easier to handle config by way of TOML files, especially for handling type (boolean, lists...).

This is a thin wrapper around the previous `udata_hydra.config` and TOML files to act like `module.VARIABLE` and avoid heavy refactoring.

Reply:
NB: toml files are recommended for configuration over ini since 3.11 https://docs.python.org/3/library/configparser.html

Reply:
NB2: the logic is similar to what we have in udata, except we don't have the convenience of `app.config.from*`.